<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbAutoSimulatorStashingComponent</class>
 <widget class="QDialog" name="CDbAutoSimulatorStashingComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cross simulator updating (FSX-P3D-FS9)</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AutoSimulator">
   <item>
    <widget class="QGroupBox" name="gb_Info">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbl_Info">
        <property name="text">
         <string>This tool checks the DB models simulator info against your local simulator info. For instance, if a model is locally found in P3D, the DB should also reflect that fact.You can easily update the sibling simulators by running the tool here. It will automatically stash the changes for you.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_UpdateModels">
     <property name="title">
      <string>Update models</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_AllSets">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rb_ScanAllModels">
        <property name="text">
         <string>scan all models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_ScanAllOrSelected</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_Selected">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_MaxModelsStashed">
        <property name="text">
         <string>Max.stashed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_SelectedOnly">
        <property name="text">
         <string>selected</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_ScanAllOrSelected</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_MaxModelsStashed">
        <property name="placeholderText">
         <string>max.models to be stashed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="swift::gui::views::CStatusMessageView" name="tvp_StatusMessages">
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pb_StashingProgress">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_OkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CStatusMessageView</class>
   <extends>QTableView</extends>
   <header>gui/views/statusmessageview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rb_SelectedOnly</tabstop>
  <tabstop>le_Selected</tabstop>
  <tabstop>rb_ScanAllModels</tabstop>
  <tabstop>le_AllSets</tabstop>
  <tabstop>tvp_StatusMessages</tabstop>
  <tabstop>le_MaxModelsStashed</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bb_OkCancel</sender>
   <signal>accepted()</signal>
   <receiver>CDbAutoSimulatorStashingComponent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_OkCancel</sender>
   <signal>rejected()</signal>
   <receiver>CDbAutoSimulatorStashingComponent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bg_ScanAllOrSelected"/>
 </buttongroups>
</ui>
